bin/linters.sh

```diff
@@ -82,6 +82,12 @@ function run_yaml_lint(){
     echo 'yamllint check passed'
 }
 
+function check_dockerfiles() {
+    echo 'Checking Dockerfiles'
+    bin/check_dockerfiles.sh
+    echo 'Dockerfiles OK'
+}
+
```

Comment:
this definitely belongs in https://github.com/istio/pkg (the common files for all repositories).

Reply:
moved in in the last commits to common-files c.f. istio/common-files#13